spacy/pipeline/spancat.py

```diff
@@ -272,6 +272,24 @@ def predict(self, docs: Iterable[Doc]):
         scores = self.model.predict((docs, indices))  # type: ignore
         return indices, scores
 
+    def suggest_candidates(
+        self, docs: Iterable[Doc], spankey: str = "candidates"
+    ) -> List[Doc]:
+        """Use the spancat suggester to add a list of span candidates to a list of docs
+
+        docs (Iterable[Doc]): The documents to predict.
```

Comment:
Agreed with Adriane's proposal to call the parameter `candidates_key` + add it to the docstring here as well.